tools/molecule.spec

```diff
@@ -0,0 +1,59 @@
+%global srcname molecule
+%global pypiname molecule
+%global pkgname python3-%{srcname}
+%global forgeurl https://github.com/ansible-community/%{srcname}
+
+Name:    %{pkgname}
+Version: 0.0.0
+Release: 1%{?dist}
+Summary: ...
+%forgemeta
+URL:       %{forgeurl}
+Source:    %{forgesource}
+License:   BSD
+BuildArch: noarch
+
+BuildRequires: python3-devel
+BuildRequires: python3-wheel
+BuildRequires: python3-tox
+# BuildRequires: python3-tox-current-env
+
+%global _description %{expand:
+...
+}
+
+%description %{_description}
+
+%py_provides python3-%{srcname}
+
+%prep
+%autosetup -p1 -n %{srcname}-%{version}
```

Comment:
Hm.. I thought this macro exists mostly for make-based projects. Need to check the docs.

Reply:
[No](https://docs.fedoraproject.org/en-US/packaging-guidelines/#_autosetup)

Reply:
Beware the RPM docs in https://rpm-software-management.github.io/rpm/manual/autosetup.html -- it has a mistake https://github.com/rpm-software-management/rpm/pull/1860